service/src/functionalTest/java/uk/gov/hmcts/reform/fpl/AdminManagesOrdersApiTests.java

```diff
@@ -0,0 +1,167 @@
+package uk.gov.hmcts.reform.fpl;
+
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import uk.gov.hmcts.reform.fpl.enums.C29ActionsPermitted;
+import uk.gov.hmcts.reform.fpl.enums.EnglandOffices;
+import uk.gov.hmcts.reform.fpl.enums.PlacedUnderOrder;
+import uk.gov.hmcts.reform.fpl.model.CallbackResponse;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.event.ManageOrdersEventData;
+import uk.gov.hmcts.reform.fpl.model.order.generated.GeneratedOrder;
+import uk.gov.hmcts.reform.fpl.service.DocumentService;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Map;
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+
+import static java.lang.String.format;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static uk.gov.hmcts.reform.fpl.enums.orders.ManageOrdersEndDateType.END_OF_PROCEEDINGS;
+import static uk.gov.hmcts.reform.fpl.enums.orders.ManageOrdersEndDateType.NUMBER_OF_MONTHS;
+import static uk.gov.hmcts.reform.fpl.model.order.Order.valueOf;
+import static uk.gov.hmcts.reform.fpl.utils.CaseDetailsHelper.formatCCDCaseNumber;
+
+import static uk.gov.hmcts.reform.fpl.utils.DateFormatterHelper.DATE;
+import static uk.gov.hmcts.reform.fpl.utils.DateFormatterHelper.DATE_TIME;
+import static uk.gov.hmcts.reform.fpl.utils.DateFormatterHelper.DATE_TIME_AT;
+import static uk.gov.hmcts.reform.fpl.utils.DateFormatterHelper.DATE_WITH_ORDINAL_SUFFIX;
+import static uk.gov.hmcts.reform.fpl.utils.DateFormatterHelper.formatLocalDateBaseUsingFormat;
+import static uk.gov.hmcts.reform.fpl.utils.DateFormatterHelper.formatLocalDateTimeBaseUsingFormat;
+import static uk.gov.hmcts.reform.fpl.utils.DateFormatterHelper.formatLocalDateToString;
+import static uk.gov.hmcts.reform.fpl.utils.DateFormatterHelper.getDayOfMonthSuffix;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.unwrapElements;
+import static uk.gov.hmcts.reform.fpl.utils.ResourceReader.readString;
+
+public class AdminManagesOrdersApiTests extends AbstractApiTest {
+
+    public static final String INPUT_FILE = "admin-manage-orders/%s/input-case-details.json";
+    public static final String EXPECTED_FILE = "admin-manage-orders/%s/expected.txt";
+    private final LocalDate todaysDate = LocalDate.now();
+    private final LocalDateTime currentDateTime = LocalDateTime.now();
+    private CaseData startingCaseData;
+
+    public final List<String> orderNumber = List.of(
+        "c32a,C32A_CARE_ORDER",
```

Comment:
wonder if we really need this as a list? maybe just adding the actual strings to the below tests would be easier to read (avoid having to index and call .split)?